docs/guides/main.rst

```diff
@@ -132,6 +132,17 @@ trouble or you have more detail to contribute here, please direct it to
 .. _install pip: https://pip.pypa.io/en/stable/installing/
 .. _get-pip.py: https://bootstrap.pypa.io/get-pip.py
 
+Installing on ARM (Raspberry Pi and similar)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Beets on ARM devices is not recommended for Linux novices. If you are
+comfortable with light troubleshooting in tools like `pip`, `make`,
```

Comment:
In RST the standard is to use two backticks for inline code / monospaced text, while single backticks are used for references (though confusingly this works fine as inline code too)

```suggestion
comfortable with light troubleshooting in tools like ``pip``, ``make``,
```